app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    @comment = Comment.create(comment_params)
+    @comment.post = post
+    @comment.author = Current.user
+    if @comment.save
+      flash[:notice] = 'Comment added successfully.'
+      redirect_to user_post_path(post.author, post)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def comment_params
+    params.permit(:text)
+  end
```

Comment:
- [ ] There should be a `new` method in the CommentsController class. Please kindly include a `new` method in this class.